client/src/ECP.ts

```diff
@@ -51,11 +51,16 @@ export class ECP {
 		this.device.setConfig(config);
 	}
 
-	public getConfig() {
+	/** Provides a way to get the whole config not just this classes' Config */
+	public getRtaConfig() {
 		if (!this.config) {
 			this.config = utils.getConfigFromEnvironmentOrConfigFile();
 		}
-		return this.config?.ECP;
+		return this.config;
+	}
+
+	public getConfig() {
```

Comment:
It might be worth adding doc comment on each of the `getConfig` methods too now that there are two methods, to help clearly call out what each of them does (an to avoid confusion with the new `getRtaConfig` method)
```suggestion
	/** 
	 * Get the ECP config from the full RTA config.
	 */
	public getConfig() {
```